json_document/tests/test_fragment_class.py

```diff
@@ -0,0 +1,130 @@
+# This file is part of json-document
+#
+# json-document is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation
+#
+# json-document is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with json-document.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for fragment classes."""
+
+from unittest2 import TestCase
+
+from json_document.document import DocumentFragment
+
+
+class DocumentFragmentClassTests(TestCase):
+    """Tests related to fragment classes."""
+
+    def test_getitem_uses_sub_value_class_from_object_property(self):
+
+        class SpecialDocumentFragment(DocumentFragment):
+            pass
+
+        fragment = DocumentFragment(
+            document=None,
+            parent=None,
+            value={
+                "item": "value"},
```

Comment:
This is copying the code from `test_misc.py`, and changing this line with break the blame of `git blame -wC` .

While I am modifying the method name, I'd prefer that the blame of the body of the method doesnt mis-attribute it to me.